duo_auth/help.md

```diff
@@ -14,26 +14,41 @@ This action is used to perform second-factor authentication.
 
 #### Options
 
-The "Options" field is used to specify additional parameters that may be necessary depending on the authentication factor selected.
-All entries use the format |Parameter|Required|Description|
+The "Options" field is used to specify additional parameters that may be necessary depending on the authentication factor selected. "Options" accepts the following parameters in json format `username`, `passcode`, `pushinfo`, `type`.
 
-##### Push
+Example:
```

Comment:
We use Example output as a convention for showing the output of actions in JSON, in most plugins. Here, let's mention this is the opposite to avoid confusion.

```suggestion
Example input:

```